news-2.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+        <head>
+            <meta charset="UTF-8">
+            <title>Virametrics - High Death Mortality Rate in Us</title>
+        </head>
+        <body>
+            <table>
+            <tr>
+            <th><h1>Virametrics</h1></th>
+            <th><img src="image/v.png"></th>
+            </tr>
+            </table> <br>
+
+            <a href="index.html">Home</a>
+            <a href="submission.html">Submit Your News</a>
+            <br>  
+            
+            <h1><b>High Death Mortality Rate in Us</b></h1>   
+            <button>Comments</button> <br>
+            <img src="image/image.png"><br>
```

Comment:
The folder name here should be `images`
```suggestion
            <img src="images/image.png"><br>
```